### PR TITLE
Fixed typo in save-artifacts description

### PR DESCRIPTION
**Pull request by soltysh**
/fyi @mfojtik

### CODE REVIEW

File: docs/builder_image.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ The `run` script is responsible for executing your application.
 ## save-artifacts
 
 The `save-artifacts` script is responsible for gathering all the dependencies which
-existence can speed up the following build processes (eg. for Ruby - Gemfiles,
+existence can speed up the following build processes (eg. for Ruby - Gemfile,
```
Comment (mfojtik) [inline docs/builder_image.md]: still not correct, in Ruby, we archive gems installed by Bundler, the Gemfile is only configuration, what we really do is we archive Gemfile.lock.... I would avoid mentioning the Gemfile at all ;-) just say for Ruby we archive gems installed by Bundler ;-)

Comment (soltysh) [inline docs/builder_image.md]: Fixed it, in both places :smile:

File: docs/builder_image.md [hunk 2]
```diff
@@ -132,3 +133,19 @@ This is a STI sample builder image, to use it, install
...
+should be following:
+
+1. Build the image.
+1. Run the image to verify `usage` script.
+1. Run `sti build` to verify `assemble` script.
+1. (optional) Run `sti build` once more to verify `save-artifacts` script and
+   `assemble`'s restore artifacts functionality.
+1. Run the image to verify the test application is working.
+
+**NOTE** The standard place to put your test application is `test/test-app` in your image repository,
```
Comment (mfojtik) [inline docs/builder_image.md]: this is only if you used sti create... the generated test/run expects the sources there... normally there is no requirement

**Comment by mfojtik** [comment]
LGTM, thanks @soltysh !

File: docs/builder_image.md [hunk 3]
```diff
@@ -132,3 +133,19 @@ This is a STI sample builder image, to use it, install
...
+should be following:
+
+1. Build the image.
+1. Run the image to verify `usage` script.
+1. Run `sti build` to verify `assemble` script.
+1. (optional) Run `sti build` once more to verify `save-artifacts` script and
+   `assemble`'s restore artifacts functionality.
+1. Run the image to verify the test application is working.
+
+**NOTE** The suggested place to put your test application is `test/test-app` in your image repository,
```
Comment (bparees) [inline docs/builder_image.md]: "to put your test application which should be built by your test/run script is..."

**Comment by bparees** [comment]
otherwise lgtm.

**Comment by soltysh** [comment]
[merge]

**Pull request merged**